hack/update-staging-godeps.sh

```diff
@@ -71,7 +71,9 @@ function updateGodepManifest() {
     go run "${KUBE_ROOT}/staging/godeps-json-updater.go" --godeps-file="${TMP_GOPATH}/src/k8s.io/${repo}/Godeps/Godeps.json" --client-go-import-path="k8s.io/${repo}"
 
     # commit so that following repos do not see this repo as dirty
-    git add vendor >/dev/null
+    # pipe through true so that command script won't error if there are no
+    # files to add
+    git add vendor/ >/dev/null || true
```

Comment:
Based on your PR #42965 that was just merged into master, it seems there is no issue bumping the godep version to v79.

As such, I've opened #42967 to bump the remaining versions to v79.